### ISSUE TITLE
Es posible crear el archivo REAME para instalar y colaborar

### ISSUE THREAD

**Issue by enlacee** [apdesc/apdesc_suc#2]
Es posible crear el archivo **README**

- La instalación paso a paso y si hay distintas instalaciónes dependiendo de las distintas versiones de la aplicación
- Esto ayuda a personas nuevas que no conoscan de VISUALBASIC pero si con las ganas de aprender y hacer. A contribuir con su grano de arena.
- Tener claro las observaciones comunes que hay que tener en cuenta si usas (LINUX, WINDOWS, MAC)
- y otras más que seguramente ustedes tendran en cuenta

Esto ayudaria a la comunidad de developer externos nacionales e internaciones a seguir contribuyendo al proyecto APDESC.

**Comment by apdesc** [comment]
Claro, gracias por la observación, estamos trabajando para crear la documentación de nuestro proyecto.
